src/utils/prompts.ts

```diff
@@ -34,6 +36,31 @@ export async function getJobInteractive(
     };
 }
 
+export async function getAllJobPostsInteractive(
```

Comment:
I used this `prompts` file to abstract some functionality that was used in two controllers, but it looks quite hacky now. We should probably use another way of sharing information (class composition?).

In the meantime, if these functions are only used in the new command, I'd have them as methods in the `RepostController`, for better cohesion.

Reply:
Agree. Thanks.